client/js/widgets/label.js

```diff
@@ -1,5 +1,37 @@
 import { Widget } from './widget.js';
 
+export async function unsetHTML(w) {
+  document.getElementById(w.id).innerHTML = null;
+	w.input = document.createElement('textarea');
+	 
+	w.addDefaults({
```

Comment:
No need to set the widget type defaults again.

Reply:
also deleted